### ISSUE TITLE
Enhance training data

### ISSUE THREAD

**Issue by dcorney** [FullFact/health-misinfo-shared#3]
### Overview

The core of this system is a fine-tuned LLM that takes as input (part of) a video transcript, and as output returns a list of health-related claims, each with a label. These labels serve two purposes: first, they can be used to indicate which claims are worth showing to fact checkers as potentially worth checking; and second they allow us to provide domain knowledge to the model during the training process. 

E.g. the claim
* Broccoli sprout extract has been shown to reduce many of the symptoms of autism in about half of the subjects compared to placebos.
is checkable because it is specific and seems to be referring to some scientific study; and it is check_worthy_ because it is suggesting a treatment for a serious condition without evidence of effectiveness.

Conversely, the claim
* Turmeric has anti-inflammatory properties that help suppress the production of fat cells.
is not really checkable because it's usually hard to prove a negative (i.e. it's hard to prove turmeric doesn't suppress fat cell production).

### Requirements

1. Develop two sets of labels, one indicating checkworthiness and one indicating not-checkworthiness. The labels should be unambiguous and easy to apply. Each set should have about 2/3/4 labels  - more than that will require large training sets.   
2. Label a set of health-related claims with these new labels. This should test out the value of the labels as well as letting us train a new, improved model

### Notes and additional information

[This is a first pass at a set of labels](https://docs.google.com/spreadsheets/d/1BNteUoyeV-l6HeCymfeSnpG6ysb4wr7axxKrpSnFFPI/). This was just created to test the workflow so needs reviewing/reworking.

[A example set of claims](https://github.com/FullFact/health-misinfo-shared/blob/main/llm_responses_natural_remedies_x5.csv), which were extracted by Gemini from a small sample of videos.

**Comment by dcorney** [comment]
Re-closing as I accidentally reopened this.
